src/main/c/src/SerialImp.c

```diff
@@ -700,16 +680,16 @@ JNIEXPORT jint JNICALL RXTXPort(open)(
 	//report_warning("\nopen() Attempting to lock: ");
 	if ( LOCK( filename, pid ) )
 	{
-		sprintf( message, "open: locking has failed for %s\n",
+		snprintf( message, sizeof(message), "open: locking has failed for %s\n",
 			filename );
 		report_error( message );
 		goto fail;
 	}
-	else
-	{
-		sprintf( message, "open: locking worked for %s\n", filename );
-		//report_warning( message );
-	}
+//	else
```

Comment:
I just saw that this block is commented. Before `report_warning()` was commented and therefore the message was never used anyways. What should we do about it? Leave it commented, remove it completely or re-introduce it with a `report()` call instead of `report_warning()`?

Reply:
I think leaving the whole block commented is fine. No point in formatting the string and then never using it.